lib/api/src/version.ts

```diff
@@ -1 +1 @@
-export const version = '5.1.0-alpha.10';
+export const version = '5.1.0-alpha.11';
```

Comment:
What's this?!

Reply:
@shilman an artifact the lib/api uses to determine the current version at runtime

It's generated in the prepare script.